shell/gpu/gpu_surface_vulkan.cc

```diff
@@ -40,7 +40,7 @@ std::unique_ptr<SurfaceFrame> GPUSurfaceVulkan::AcquireFrame(
     }
     return weak_this->window_.SwapBuffers();
   };
-  return std::make_unique<SurfaceFrame>(std::move(surface),
+  return std::make_unique<SurfaceFrame>(std::move(surface), true,
```

Comment:
Need to verify that this is the appropriate value for Vulkan surfaces.

Reply:
This appears to be correct as we specify no clipping when constructing the swapchain in vulkan_swapchain.cc:

```
133:      .clipped = VK_FALSE,
```

According to Vulkan documentation, clipped=TRUE allows them to drop pixels for parts of the surface that are hidden behind other windows or otherwise clipped out and the application should set this to FALSE if they expect to read back the pixels.